gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ProjectInfo.java

```diff
@@ -115,7 +115,66 @@ static ResourceId fromPb(
     }
   }
 
-  public static class Builder {
+  public static abstract class Builder {
```

Comment:
I just noticed this: is there a reason for this to be an abstract class rather than an interface?